src/components/common/button-icon/button-icon.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { VFC } from 'react';
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons';
+import { TouchableOpacity } from 'react-native';
+import globalStyles from '~global/constants.style';
+
+export const ButtonIcon: VFC<Props> = ({ onPress, variant }) => (
+  <TouchableOpacity onPress={onPress}>
+    {variant === 'add' ? (
```

Comment:
lets find some solution how we can change ifs to switch here

Reply:
changed to switch